catalog/pipeline/fbc-release/0.1/fbc-release.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: bundle-fbc-release-pipelinerun
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton release pipeline to interact with FBC Pipeline
+  params:
+    - name: snapshot
+      type: string
+      description: The Snapshot in JSON format
+    - name: enterpriseContractPolicy
+      type: string
+      description: JSON representation of the EnterpriseContractPolicy
+    - name: fbcFragment
+      type: string
+      description: FBC fragment built by HACBS
+    - name: fromIndex
+      type: string
+      description: Index image (catalog of catalogs) the FBC fragment will be added to
+    - name: overwriteFromIndex
+      type: string
+      description: Boolean indicating if the from_index should be overwritten
+    - name: binaryImage
+      type: string
+      description: OCP binary image to be baked into the index image.
```

Comment:
unnecessary period at end of line

Reply:
TBH I followed what I saw in the release.yaml for enterprise contract params. As it was a copy+paste keeping the EC part the parameters came in uppercase, hence I kept the same way for the new task. I don't have a strong reason to keep it, but after they were there thought it somehow gave a clear vision that they are parameters for the task and used in their script, and as normally shell vars are uppercase I thought they would fit properly.

I am up to change it if I am saying no sense :D @johnbieren

Reply:
I think we have been doing it `likeThisForExample` because we sometimes pass our parameters like this https://github.com/redhat-appstudio/release-service/blob/main/tekton/pipeline_run.go#L111 which wouldn't work with all uppercase. We could of course change the controller code to put the parameter all uppercase. I don't have a strong preference as long as we are consistent with our pipelines and tasks. Maybe change the parameters in this PR to be our style and we can discuss with @davidmogar once he is back from pto if we should make a ticket to change all our tekton resource parameters to uppercase as it can make it clear they are parameters and follows shell standards